steps/pagerduty/0.1.0/step.yml

```diff
@@ -0,0 +1,109 @@
+title: PagerDuty
+summary: |
+  Create a PagerDuty alert
+description: |
+  This step creates a PagerDuty alert with specific parameters. It can use Events API v1 or v2.
+website: https://github.com/PagerDuty/bitrise-step-pagerduty
+source_code_url: https://github.com/PagerDuty/bitrise-step-pagerduty
+support_url: https://github.com/PagerDuty/bitrise-step-pagerduty/issues
+published_at: 2019-09-13T14:04:48.789894-04:00
+source:
+  git: https://github.com/PagerDuty/bitrise-step-pagerduty.git
+  commit: f75718ef2c7fa0c1e0af462733585d8c51bffddc
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  - name: jq
+  apt_get:
+  - name: git
+  - name: wget
+  - name: curl
+  - name: jq
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- api_version: v2
+  opts:
+    description: Events API v1 or v2 (available in the Integrations tab of the Service
+      Details).
+    is_expand: false
+    is_required: true
+    summary: Events API v1 or v2 (available in the Integrations tab of the Service
+      Details)
+    title: API version
+    value_options:
+    - v1
+    - v2
+- integration_key: null
+  opts:
+    description: The hexadecimal service key (available in the Integrations tab of
+      the Service Details).
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: The hexadecimal service key (available in the Integrations tab of the
+      Service Details)
+    title: Integration key
+    value_options: []
+- description: null
+  opts:
+    description: The description that the event will be created with.
+    is_expand: true
+    is_required: true
+    summary: The description that the event will be created with
+    title: Event description
+    value_options: []
+- opts:
+    description: The severity that the event will be created with (relevant to Event
+      Rules).
+    is_expand: false
+    is_required: false
+    summary: The severity that the event will be created with (relevant to Event Rules)
+    title: Event severity (API v2 only)
+    value_options:
+    - critical
+    - error
+    - warning
+    - info
+  severity: error
```

Comment:
It looks like this got mangled... it should be:

```
- severity: error
    opts:
      stuff
```

Reply:
See https://github.com/PagerDuty/bitrise-step-pagerduty/blob/master/step.yml#L77